rocky/rocky/views/upload_raw.py

```diff
@@ -62,7 +62,7 @@ def process_raw(self, form):
 
         try:
             get_bytes_client(self.organization.code).upload_raw(raw_file.read(), mime_types)
-        except HTTPError as e:
+        except HTTPStatusError as e:
```

Comment:
The old code was wrong, because we should also handle other errors that can happen, so need to catch `HTTPError`.

Reply:
You added the isinstance check but forgot to change the except clause.